editor/filesystem_dock.cpp

```diff
@@ -2978,7 +2978,7 @@ void FileSystemDock::_file_sort_popup(int p_id) {
 MenuButton *FileSystemDock::_create_file_menu_button() {
 	MenuButton *button = memnew(MenuButton);
 	button->set_flat(true);
-	button->set_tooltip(TTR("Sort files"));
+	button->set_tooltip(TTR("Sort Files"));
```

Comment:
If it's a tooltip, it should use sentence case, no?

Reply:
There are several tooltips in the editor that are written in title case. This one is for a `MenuButton`, so I think a title fits here as well.